client/src/components/page/About.js

```diff
@@ -55,7 +55,7 @@ class About extends Component {
 }
 
 About.propTypes = {
-  contributors: PropTypes.arrayOf(PropTypes.instanceOf(Contributor))
+  contributors: PropTypes.object
```

Comment:
Note: There was a runtime error being thrown because the proptype isn't an array. It is actually a json object.